### PR TITLE
123 - Add On Chain Voting Helm Chart

### PR DESCRIPTION
**Pull request by kaozenn**

### CODE REVIEW

File: on-chain-voting/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+Or use helmfile only to generate resources and apply them with kubectl like so:
+
+```bash
+$ helmfile template | kubectl apply -f -
+```
+
+## Configuration
+
+To get all available values in cloned `mina-helm-charts-private` do:
```
Comment (simisimis) [inline on-chain-voting/README.md:42]: This is now renamed to helm-charts

File: on-chain-voting/README.md
Comment (simisimis) [inline on-chain-voting/README.md:1]: web as a name feels a bit off. Maybe backend and frontend would better describe roles?

**Review by simisimis — APPROVED** [review]
If you prefer web/server over frontend/backend, feel free to merge it.

File: on-chain-voting/README.md
Comment (kaozenn) [reply on-chain-voting/README.md:1]: I did not choose this terminology
- https://github.com/MinaFoundation/mina-on-chain-voting/tree/mf-main/server
- https://github.com/MinaFoundation/mina-on-chain-voting/tree/mf-main/web

File: on-chain-voting/README.md (hunk 1 above)
Comment (kaozenn) [reply on-chain-voting/README.md:42]: Good catch, I have updated the charts

File: on-chain-voting/README.md
Comment (simisimis) [reply on-chain-voting/README.md:1]: ha, good that I approved it then 🫣

**Pull request merged**